packages/react-reconciler/src/__tests__/ReactUse-test.js

```diff
@@ -561,6 +577,7 @@ describe('ReactUse', () => {
           'triggered by `use`, wrap your component in a error boundary.',
       );
     }
+    console.error.mockRestore();
```

Comment:
If you don't restore the mock at the end of a test using `spyOnDev`, all tests after it will have the console errors silenced and break the `toErrorDev` / `assertConsoleErrorDev` tests. 

This quick fix isn't the best either because if this test fails, the mock won't restore so test after it will fail even though they pass. The best would be to restore in an `afterEach` in `setupTests.js`, but that will fail a bunch of tests so I'll follow up with that.